gateway/src/apicast/configuration_loader.lua

```diff
@@ -229,7 +235,7 @@ local function lazy_load_config(configuration, host)
     if not config then
       ngx.log(ngx.WARN, 'failed to get config for host: ', host)
     end
-    _M.configure(configuration, config)
+    _M.configure(configuration, config, host)
```

Comment:
`host` ? shouldn't be `true`?

Reply:
Removed. Lazy mode will not return the stale configuration so we don't really need to clear the cache.